pom.xml

```diff
@@ -56,6 +59,81 @@
             </snapshots>
         </repository>
     </repositories>
+    <dependencies>
```

Comment:
I'm actually not a big fan of listing the dependencies in the parent in the `dependencies` section, as they are inherited by all children, when they don't actually need it.
A better practice is to use the `dependencyManagement.dependencies` section of the parent pom to define the versions of those dependencies, but let each child explicitly defines the dependencies they need (but omitting the version as already set in the parent)

Reply:
I agree on that, I will move them back to their dedicated modules